lib/config/migration.ts

```diff
@@ -441,6 +447,20 @@ export function migrateConfig(
         }
       }
     }
+    if (is.array(migratedConfig.regexManagers)) {
+      const renameMap = {
+        lookupNameTemplate: 'packageNameTemplate',
```

Comment:
Why not using the new rename migration?

https://github.com/renovatebot/renovate/blob/00cab7c3a7f9abb09a16bdc34c9681ed3bd04fea/lib/config/migrations/migrations-service.ts#L42-L50